compiler/src/dotty/tools/dotc/core/tasty/TreeUnpickler.scala

```diff
@@ -531,7 +531,7 @@ class TreeUnpickler(reader: TastyReader,
         def complete(denot: SymDenotation)(implicit ctx: Context) =
           denot.info = typeReader.readType()
       }
-      val sym = ctx.newSymbol(ctx.owner, name, Flags.Case, completer, coord = coordAt(start))
+      val sym = ctx.newSymbol(ctx.owner, name, Flags.Case, completer, coord = coordAt(start)(ctx.addMode(Mode.ReadPositions)))
```

Comment:
`addMode` is somewhat expensive since it will require making a copy of the Context if the mode isn't already present. Maybe `coordAt` should have an extra parameter to force reading the position even if ReadPosition isn't set. I think we should discuss the purpose of ReadPosition in the next meeting because I don't recall the details.